src/App.vue

```diff
@@ -124,6 +124,17 @@ export default Vue.extend({
             })
         },
         type: 'checkbox-filter'
+      }, {
+        name: 'boolean-filter',
+        label: 'Boolean filter',
+        collapsed: true,
+        options: () => {
+          return new Promise(
```

Comment:
I don't think that the labels are fields that you would want to request dynamically at a later time. I think this is beter served as direct properties